packages/core/src/helpers.ts

```diff
@@ -25,12 +27,72 @@ export const formatDuration = (duration: number) => {
     }${milliseconds ? `${milliseconds}ms` : ''}`.trim();
 };
 
-export const getResolvedPath = (filepath: string) => {
-    try {
-        return require.resolve(filepath);
-    } catch (e) {
-        return filepath;
+// https://esbuild.github.io/api/#glob-style-entry-points
+const getAllEntryFiles = (filepath: string): string[] => {
+    if (!filepath.includes('*')) {
+        return [filepath];
```

Comment:
I don't think this is enough to know if this is a valid path / something that has to be globed. For instance `hello.{js,ts}` has to go through `glob`